components/Flight.tsx

```diff
@@ -65,7 +66,9 @@ function flightRenderer(flight: IFlight) {
   const departureDay = convertDateToDay(departureDate).split(' ').join('\n');
   const departureTime = convertDateToTime(departureDate);
   const arrivalTime = convertDateToTime(new Date(arrival));
-  const flightLength = lengthOfFlight.split(' ')[0].split(':').join('h ') + 'm';
+  const flightLength = lengthOfFlight; //.split(' ')[0].split(':').join('h ') + 'm';
```

Comment:
Has the flight time changed in the back-end? Because if not, the h/m addition makes it more user friendly.

Reply:
lengthOfFlight is PT1H45M in backhend, and you had another format in the mock, so yes

Reply:
did it in a probably wonky way but it's not urgent so it works. 
//to refactor